proteuslib/unit_models/reverse_osmosis_0D.py

```diff
@@ -1045,13 +1046,69 @@ def initialize(blk, initialize_guess=None, state_args=None, outlvl=idaeslog.NOTS
         )
 
     def _get_performance_contents(self, time_point=0):
-        # TODO: make a unit specific stream table
         var_dict = {}
+        var_dict["Volumetric Recovery Rate"] = self.recovery_vol_phase[time_point, 'Liq']
+        var_dict["Solvent Mass Recovery Rate"] = self.recovery_mass_phase_comp[time_point, 'Liq', 'H2O']
+        var_dict["Membrane Area"] = self.area
+        if hasattr(self, "length"):
+            var_dict["Membrane Length"] = self.length
+        if hasattr(self, "width"):
+            var_dict["Membrane Width"] = self.width
         if hasattr(self, "deltaP"):
-            var_dict["Pressure Change"] = self.deltaP[time_point]
+            var_dict["Pressure Drop"] = self.deltaP[time_point]
```

Comment:
This is very pedantic -- but isn't "Pressure Change" more correct, as this reports a negative quantity. That is, "Pressure Drop" would be the negative of `self.deltaP[time_point]`.

Reply:
I agree -- I'll change it back.